api/db-cdr/generate-cdr/bq-schemas/person_gender_identity.json

```diff
@@ -0,0 +1,17 @@
+[
```

Comment:
This is an extra intermediate table we're writing to BigQuery, right?

Reply:
yeah, so basically we want to also show the gender identity graph along with the biological sex pie.